lib/oauther.ex

```diff
@@ -94,7 +100,7 @@ defmodule OAuther do
   end
 
   defp nonce do
-    :crypto.rand_bytes(32) |> Base.encode64
+    :crypto.rand_bytes(16) |> Hexate.encode
```

Comment:
I can understand that, it does seem lame to have to add an extra package for one (non spec compliant) API. Maybe it would be better to make it possible to specify a nonce generation function? I'll see if I can look into that